include/z64.h

```diff
@@ -279,30 +280,25 @@ typedef struct GraphicsContext {
 } GraphicsContext; // size = 0x300
 
 typedef struct {
-    /* 0x00 */ union {
-        struct  {
-            u16 a   : 1;
-            u16 b   : 1;
-            u16 z   : 1;
-            u16 s   : 1;
-            u16 du  : 1;
-            u16 dd  : 1;
-            u16 dl  : 1;
-            u16 dr  : 1;
-            u16     : 2;
-            u16 l   : 1;
-            u16 r   : 1;
-            u16 cu  : 1;
-            u16 cd  : 1;
-            u16 cl  : 1;
-            u16 cr  : 1;
+    PadInput input;
+    union{
+        u16 status;
+        struct{
+            u8 errno;
+            u8 status_lo;
         };
-        u16        pad;
     };
-    /* 0x02 */ s8  x;
-    /* 0x03 */ s8  y;
-} RawInput; // size = 0x4
+} PadState;
 
+typedef struct
+{
+    /* 0x00 */ PadState current;
+    /* 0x06 */ PadState prev;
+    /* 0x0C */ PadState pressed_diff;
+    /* 0x12 */ PadState released_adj;
```

Comment:
To fit with our naming scheme, struct members should use camelCase like variables.
```suggestion
    /* 0x0C */ PadState pressedDiff;
    /* 0x12 */ PadState releasedAdj;
```